appeals/web/src/server/lib/mappers/components/date.js

```diff
@@ -0,0 +1,107 @@
+import { dateISOStringToDisplayDate } from '#lib/dates.js';
+import { kebabCase } from 'lodash-es';
+
+/**
+ *
+ * @param {Object} options
+ * @param {string} options.id
+ * @param {string} options.text
+ * @param {string|null} [options.value]
+ * @param {string} [options.defaultText]
+ * @param {string} options.link
+ * @param {boolean} [options.addCyAttribute]
+ * @param {boolean} options.editable
+ * @param {string} [options.classes]
+ * @returns {Instructions}
+ */
+export function dateSummaryListItem({
```

Comment:
Love that we're seeing more of these appear! I've just put in an instructions dir in this components dir, would you mind moving this into there please?

Reply:
Done - sorry, I didn't think to update this after I rebased main into my branch! It's nice to have the separation between instructions and page components too :)